src/nunavut/lang/c/templates/_fields.j2

```diff
@@ -0,0 +1,11 @@
+{%- for field in composite_type.fields %}
+{%- if field is not padding %}
+    {{ field.data_type | declaration }} {{ field | id }}
+    {%- if field.data_type is ArrayType -%}
+        [{{ field.data_type.capacity }}]
+    {%- endif -%};
+{%- if field is VariableLengthArrayType %}
+    {{ typename_unsigned_length }} {{ field | id }}_length;
```

Comment:
I spent a considerable amount of time trying to find the source that advised to put length *before* the array in structures and parameter list declarations but I couldn't find it. I recall reading that some time ago possibly in the context of C11. Can we please put the length before the array? I might find the source later.

Reply:
pish-posh. Okay